src/pages/ReportSettingsPage.js

```diff
@@ -180,33 +166,18 @@ class ReportSettingsPage extends Component {
                                                     {this.props.translate('newRoomPage.roomName')}
                                                 </Text>
                                                 <Text numberOfLines={1} style={[styles.optionAlternateText]}>
-                                                    {this.state.newRoomName}
+                                                    {this.props.report.reportName}
                                                 </Text>
                                             </View>
                                         )
                                             : (
                                                 <RoomNameInput
-                                                    ref={el => this.roomNameInputRef = el}
-                                                    value={this.state.newRoomName}
-                                                    policyID={linkedWorkspace && linkedWorkspace.id}
-                                                    errorText={this.state.errors.newRoomName}
-                                                    onChangeText={newRoomName => this.clearErrorAndSetValue('newRoomName', newRoomName)}
+                                                    inputID="newRoomName"
+                                                    defaultValue={this.props.report.reportName}
                                                     disabled={shouldDisableRename}
```

Comment:
We can remove `disabled={shouldDisableRename}`, and `shouldDisableRename` will not show the `RoomNameInput`, no?

Reply:
yep that make sense